backend/src/main/kotlin/com/infowings/catalog/data/objekt/ObjectPropertyValue.kt

```diff
@@ -101,4 +103,62 @@ data class ObjectPropertyValue(
     val aspectProperty: AspectPropertyVertex?,
     val parentValue: ObjectPropertyValueVertex?,
     val measure: OVertex?
-)
\ No newline at end of file
+)
+
+data class ValueResult(
+    private val valueVertex: ObjectPropertyValueVertex,
+    val valueDto: ValueDTO,
+    val measureId: String?,
+    private val objectProperty: ObjectPropertyVertex,
+    private val aspectProperty: AspectPropertyVertex?,
+    private val parentValue: ObjectPropertyValueVertex?
+) {
+    val id: String
+        get() = valueVertex.id
```

Comment:
that makes sense only if `valueVertex.id` is mutable but it looks like it's not
If `valueVertex` is immutable would be better this: 
```kotlin
 val id: String = valueVertex.id
```

Reply:
Yep, will change.

Reply:
Actually the thing is: ID is kind of mutable. It is mutated when the vertex leaves transaction boundary and thus is saved. In that case ID changes from temporary one to persistent that is assigned to the vertex. The same thing is with version: it is incremented when vertex successfully leaves transaction.